cmd/mattermost/commands/webhook.go

```diff
@@ -102,33 +101,26 @@ func listWebhookCmdF(command *cobra.Command, args []string) error {
 		}
 
 		// Fetch all hooks with a very large limit so we get them all.
-		incomingResult := make(chan store.StoreResult, 1)
```

Comment:
You are removing here the explicit async approach, we want to request this in an async way, see the example for this cases